<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVehicleFormWindow</class>
 <widget class="QDialog" name="AddVehicleFormWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Añadir Vehículo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="wheelNumLabel">
       <property name="text">
        <string>N.º de ruedas:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="engineLabel">
       <property name="text">
        <string>Motor:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="engineCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="engineHpLabel">
       <property name="text">
        <string>Potencia del motor (cv):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="engineHpSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>80</number>
       </property>
       <property name="maximum">
        <number>450</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="fuelLabel">
       <property name="text">
        <string>Combustible:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="fuelCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="fuelTypeLabel">
       <property name="text">
        <string>Tipo de combustible:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="fuelTypeComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Eléctrico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gasolina</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diésel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Híbrido</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Queroseno</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="colorComboBox">
       <item>
        <property name="text">
         <string>Blanco</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Negro</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rojo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Azul</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Amarillo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Verde</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="wingsLabel">
       <property name="text">
        <string>Alas:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="wingsCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="reactorsLabel">
       <property name="text">
        <string>Reactores:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="reactorsCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="undercarriageLabel">
       <property name="text">
        <string>Tren de aterrizaje:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="locomotiveLabel">
       <property name="text">
        <string>Locomotora:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QComboBox" name="repairKitComboBox">
       <item>
        <property name="text">
         <string>Rueda de repuesto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kit de reparación de pinchazos</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="undercarriageCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="locomotiveCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="wagonNumLabel">
       <property name="text">
        <string>N.º de vagones:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="wagonNumSpinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="wheelNumComboBox">
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="repairKitLabel">
       <property name="text">
        <string>Kit:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="plateLabel">
       <property name="text">
        <string>Matrícula:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="plateLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="generatePlateBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Generar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addVehicleBtn">
       <property name="text">
        <string>Añadir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>engineCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>engineHpSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fuelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fuelTypeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
